src/pages/priority/components/RegisterPriorityForm.tsx

```diff
@@ -1,67 +0,0 @@
-import { Form, FormInstance, Input, InputNumber } from "antd";
-import Strings from "../../../utils/localizations/Strings";
-import { BsCardText } from "react-icons/bs";
```

Comment:
Why did you delete this?

Reply:
That file is not more in this PR, it is a file from KAN-12 (priority) I fixed the PR doing the rebase as we talked. Let me know if I have to do any other thing to fix the PR, I just see the 4 files changed.